include/packetgraph/seccomp-bpf.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * seccomp example for x86 (32-bit and 64-bit) with BPF macros
+ *
+ * Copyright (c) 2012 The Chromium OS Authors <chromium-os-dev@chromium.org>
+ * Authors:
+ *  Will Drewry <wad@chromium.org>
+ *  Kees Cook <keescook@chromium.org>
+ *
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
```

Comment:
Which LICENSE File ?
I guess you should add this file, maybe call it seccomp-bpf-licence
and then change this comment